docs/data-sources/rule.md

```diff
@@ -8,14 +8,23 @@ page_title: "Coralogix: coralogix_rule"
 Use this data source to retrieve information about a Coralogix Rule.
 
 ## Example Usage
-
+once a rule group is created it can be refered to in your config.
+This example describes the data structurre of the rule
 ```hcl
-data "coralogix_rule" "rule" {
+data "coralogix_rule" "my_rule" {
     rule_id        = "e1a31d75-36ab-11e8-af8f-02420a00070c"
     rules_group_id = "e10ef9d1-36ab-11e8-af8f-02420a00070c"
 }
 ```
 
+Using a code exmple like this will output the id of suce a rule:
```

Comment:
exmple --> example,

id of suce a rule --> what are you trying to say here?

Reply:
Fixed Type-o.
we have been getting the question of how do "debug print" the attrib. of the resources that we created.
so i added this example to demonstrate both the output function and the var based pointer.

Reply:
@oded-dd ^